tutorials/shading/shading_reference/shading_language.rst

```diff
@@ -252,6 +252,22 @@ Arrays also have a built-in function ``.length()`` (not to be confused with the
 
 Note: If you use an index below 0 or greater than array size - the shader will crash and break rendering. To prevent this, use ``length()``, ``if``, or ``clamp()`` functions to ensure the index is between 0 and the array's length. Always carefully test and check your code. If you pass a constant expression or a simple number, the editor will check its bounds to prevent this crash.
 
+Global arrays
+~~~~~~~~~~~~~
+
+You can declare arrays at global space like:
+
+.. code-block:: glsl
+    shader_type spatial;
+
+    const lowp vec3 v[1] = lowp vec3[1] ( vec3(0, 0, 1) );
+
+    void fragment() {
+      ALBEDO = v[0]; 
+    }
+
+The only difference in syntax from local shader arrays is the required constness.
```

Comment:
```suggestion
.. note: 
    Global arrays have to be declared as global constants, otherwise they can be declared the same as local arrays.
```

Reply:
Changed. Note: At global level they cannot be declared without const.